server/src/callbacks.mjs

```diff
@@ -37,6 +37,8 @@ function validateURL(url) {
 
 Empirica.onGameStart(function ({ game }) {
   console.log("game start");
+  console.log("game." + game.batch.get("TVSurveys")[game.treatment.TVSurvey] != null)
```

Comment:
this is a bit of an ambiguous message, isn't it? "game.true"?

Reply:
oops, I meant to delete that it was for debugging